vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -17,11 +17,25 @@ def ems_inv_to_hashes(inventory)
       get_pods(inventory)
       get_endpoints(inventory)
       get_services(inventory)
+      get_registries
+      get_images
 
       EmsRefresh.log_inv_debug_trace(@data, "data:")
       @data
     end
 
+    def get_images
+      images = @data_index.fetch_path(:container_image, :by_ref_and_registry_host_port).try(:values) || []
+
```

Comment:
Minor: no need for extra empty line, we're going to `process_collection` the `images` we found here.